DOC-1932: Updates to reflect changes to CENM User Admin Tool UI (#444)
corda/corda-docs: content/en/docs/cenm/1.5/user-admin.md

### Who can use it

> You need administrator access to the User Admin tool to use it. If you are the first person using the tool in your organisation, you need to request access from your network owner's organization.

Reviewer: ```suggestion
You need administrator access to use the User Admin tool. If you are the first person using the tool in your organisation, you need to request access from your network owner's organisation.
```

## Access the CENM User Admin tool

> Access the User Admin tool from your [Gateway service](gateway-service#manage-your-configuration) instance. Enter the full address of your Gateway service, including the port number, followed by `/admin` into a web browser.

Reviewer: ```suggestion
Access the User Admin tool from your [Gateway Service](gateway-service#manage-your-configuration) instance. Enter the full address of your Gateway Service, including the port number, followed by `/admin` into a web browser.
```

To change your password:

> 1. Login to the User Admin tool. If this is your first login or you have requested a new login, use the temporary password you have been given.

Reviewer: ```suggestion
1. Log in to the User Admin tool. If this is your first login or you have requested a new login, use the temporary password you have been given.
```

You have changed your password. You are automatically logged out of the User Admin tool, and can now log back in using your new password.

## Create a new user or administrator

Users can access network services to perform tasks. To give a user permissions, you can assign them a role, and/or add them to a group with set permissions.

Administrators *cannot* have any role as a user on your network operation services.

![create_user](resources/create_user.png)

{{< note >}}
You must be registered as an administrator to create new users and administrators.
{{< /note >}}

To add a new user or administrator:

1. Select **Users** from the side menu.

2. Click **Create User**.

3. Fill in the **New User Form**.
    a. Enter a username (required).
    b. Add an optional display name and email.

>     b. If you would like to add the user to any groups, select them from the dropdown menu.

Reviewer: ```suggestion
    b. If you would like to add the user to any groups, select them from the drop-down menu.
```

Author: The MS style guide wants this as one word, so I'll leave it for now.

    c. Enter and confirm a temporary password - the user can change this when they log in for the first time.

4. If you are creating an administrator, set the **Admin** toggle to 'on'.

5. Set the **Enabled** toggle to 'on' to activate the user. You can return to this step later if you do not need to activate the user yet.

8. Click **Create User** to complete the registration.

9. Send the user their login details. Remind them to change their password.

## Manage a user

You can change a user's status, password, or group membership from the **User Details** screen.

![Manage user](resources/edit_user.png)

1. Select **Users** from the side menu to view the user list.

2. Click the **Username** of the relevant user.

3. Change a user's status or delete a user from the **User Details** panel:
    * **Locked** toggle: If a user tries an incorrect password too many times, they will be locked. Click the **Locked** toggle to unlock them and allow additional attempts.

>     * **Admin** toggle: activate the **Admin** toggle to make a user an administrator.

Reviewer: ```suggestion
    * **Admin** toggle: Activate the **Admin** toggle to make a user an administrator.
```

>     * **Enable** toggle: deactivate or reactivate a user.

Reviewer: ```suggestion
    * **Enable** toggle: Deactivate or reactivate a user.
```

>     * **Delete User** button: remove the user from the system.

Reviewer: ```suggestion
    * **Delete User** button: Remove the user from the system.
```

    Updates save automatically.

4. Reset passwords from the **Change Password** panel. You are required to enter your own password before you can change another user's.

5. Change a user's group membership from the **Change Groups** panel.

## Create a group

Groups let you grant multiple users the same set of permissions. Groups make it easier for you to manage permissions of future users - just add them to the relevant groups instead of configuring individual roles.

![Creating a group](resources/create_group.png)

To create a group:

1. Select **Groups** from the side menu. You can see any existing groups.

2. Click **Add Group**.

3. Fill in the **New Group Form**.
    a. Enter a group name (required).
    b. Add a description (optional).

>     c. Select group members from the dropdown.

Reviewer: ```suggestion
    c. Select group members from the drop-down menu.
```

    d. Toggle the **Admin** option on to make all group members administrators.
    e. Toggle the **Auto-Enroll** option on to assign this group to all new users you create.

4. Click **Create Group**.

You can access all your groups from the **Groups** screen.

## Manage a group

You can add or remove members of a group, or delete an existing group. Deleting a group does not delete the users in the group.

![Manage a group](resources/editing_group.png)
To make changes to a group:

1. Select **Groups** from the side menu to view your existing groups.

2. Click the relevant group.

3. Update your group from the  **Group details** panel:

>     * **Pencil icons** edit the name and description of the group.

Reviewer: ```suggestion
    * **Pencil icons**: Edit the name and description of the group.
```

>     * **Enable** toggle: activate or deactivate the auto-enrolment of new users into the group.

Reviewer: ```suggestion
    * **Enable** toggle: Activate or deactivate the auto-enrolment of new users into the group.
```

>     * **Admin** toggle: set the administrator status of the group.

Reviewer: ```suggestion
    * **Admin** toggle: Set the administrator status of the group.
```

    * **Delete button**: Delete the group. This does not delete the users in the group - they are still active, but do not have any permissions that were assigned at the group level.

    Changes save automatically.

> 4. Edit the users in a group from the 'Edit user assignment' panel.

Reviewer: ```suggestion
4. Edit the users in a group from the **Edit user assignment** panel.
```

>     * Open the dropdown to see a list of all users.

Reviewer: ```suggestion
    a. Click the drop-down menu to see a list of all users.
```

>     * Tick the box next to a user to add them to the group.

Reviewer: ```suggestion
    b. Select the checkbox next to a user to add them to the group.
```

>     * Click the 'x' next to a user's name to remove them from the group. This will revoke any permissions associated with group membership - you can confirm this by checking the user's individual roles.

Reviewer: ```suggestion
    c. Click the **x** next to a user's name to remove them from the group. This will revoke any permissions associated with group membership - you can confirm this by checking the user's individual roles.
```

>     * Click **Save**.

Reviewer: ```suggestion
    d. Click **Save**.
```
